pxe/common_files/disk_imaging.sh

```
@@ -217,22 +217,149 @@ function resolve_esp_partition() {
+
+	devices_out=()
+	blkid_output=$(blkid -c /dev/null -t "$identifier_type=$identifier" -o device 2>&1)
+	blkid_status=$?
+
+	case "$blkid_status" in
+		0)
+			if [ -z "$blkid_output" ]; then
+				echo "blkid returned success without a device for $identifier_type=$identifier" | tee "$log_output" >&2
+				return 1
+			fi
+			mapfile -t devices_out <<< "$blkid_output"
```

Shellcheck (0.11.0)

[warning] devices_out appears unused. Verify use (or export if used externally).

(SC2034)